webpack.config.js

```diff
@@ -19,9 +19,9 @@ module.exports = (env, argv) => {
           tag: packageJSON.version
         },
         buildInfoString = `${buildInfo.date} ${buildInfo.tag}`,
-        cfmUrl = devMode ? "/cfm" : "https://cloud-file-manager.concord.org/branch/master",
-        codapUrl = devMode ? "/codap/static/dg/en/cert/index.html" : "/~dmartin/travis4/static/dg/en/cert/index.html",
-        sageUrl = devMode ? "/sage" : "https://sage.concord.org/branch/master";
+        cfmUrl = "/cfm",
+        codapUrl = devMode ? "/codap/static/dg/en/cert/index.html" : "/releases/staging/static/dg/en/cert/index.html",
```

Comment:
Is `"/releases/staging/static/dg/en/cert/index.html"` correct? Aren't we missing `/codap` here?

Reply:
No, its `/releases` because of the way CloudFront works - by default it does not remove the prefix used to match the redirection (called "behaviors" in CloudFront) so `sagemodeler.com/releases` maps to `codap-server.concord.org/releases`.  I've added a Lamda function to the `/cfm` and `/sage` behaviors that does remove the first folder in the path so `sagemodeler.concord.org/cfm` maps to `model-resources/cloud-file-manager`, if I didn't it would map to `model-resources/cloud-file-manager/cfm`